tests/WalletMgr.py

```diff
@@ -12,9 +12,9 @@
 Wallet=namedtuple("Wallet", "name password host port")
 # pylint: disable=too-many-instance-attributes
 class WalletMgr(object):
```

Comment:
We shouldn't be using `findAvailablePort` since the port numbers that keosd is allowed to use are strongly constrained now. We should be using a range of ports that are hopefully not taken on the machines these tests run on. If so, as long as two tests don't run with the same cluster ID, the hope is that the assigned port number could be used without conflict. If there is a conflict, then the test must shutdown. So this will require changes to the `launch` function to not use `findAvailablePort` and furthermore, the port should always be specified to the `WalletMgr`.

Reply:
will remove findAvailablePort

Reply:
it can be specified, or by default the port is ```calculate_port(cluster_id, 1, port_type::keosd)```